packages/next/babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  presets: ['next/babel'],
+  plugins: [
+    ['react-native-web', { commonjs: true }],
+    ['transform-define', {
+      '__DEV__': process.env.NODE_ENV,
+      '__SUBPLATFORM__': 'next',
```

Comment:
Great!

Reply:
For some reason I wasn't able to define the in Next's webpack config but this ended up working out nicely.